apps/files/src/fileactions.js

```diff
@@ -40,6 +40,14 @@ export default {
             }
             return item.canBeDeleted()
           }
+        },
+        {
+          icon: 'star',
+          handler: this.toggleFileFavorite,
+          ariaLabel: this.$gettext('Mark/Unmark as favorite'),
```

Comment:
@LukasHirt Any idea how to just use 'mark as favorite' or 'Unmark as favorite'. I couldn't find any possibility to access the file.starred value in this context

Reply:
You could transform `ariaLabel` into a method which accepts resource as an argument. Same as is done with `isEnabled`

Reply:
The problem: When trying to do that the function gets interpreted as string value

Reply:
done